clouddriver-lambda/src/main/java/com/netflix/spinnaker/clouddriver/lambda/service/LambdaService.java

```diff
@@ -273,33 +239,26 @@ private Void addTargetGroupAttributes(
     return null;
   }
 
+  private static final Predicate<Statement> isLambdaInvokeAction =
+      statement ->
+          statement.getActions().stream()
+              .anyMatch(action -> action.getActionName().equals("lambda:InvokeFunction"));
+  private static final Predicate<Statement> isElbPrincipal =
+      statement ->
+          statement.getPrincipals().stream()
+              .anyMatch(
+                  principal -> principal.getId().equals("elasticloadbalancing.amazonaws.com"));
```

Comment:
does it make sense to inverse the equals to avoid a possible NPE `"elasticloadbalancing.amazonaws.com".equals(principal.getId())` ?

Reply:
Probably wasn't super critical - I think ID was always there, but done ;)